### DISCUSSION TITLE
Try It: Selection Feature for Adding query parameters back

### DISCUSSION THREAD

**Discussion by nosvalds** [Azure/api-management-developer-portal#2594] [Feature request]
## Enhancement description

In the Try It sidebar it would be nice if after you remove some query parameters, when you go to "Add parameter" it gives you a selection dropdown of the available query parameters.

This would be useful for APIs with a lot of query parameters. So the user doesn't have to go back to the documentation to figure out which parameters are available.

## Reproduction steps

1. Go to an API in the Developer Portal
2. Click on "Try It"
3. Scroll down to "Parameters"
4. Delete a Query Parameter5. 
5. Scroll down and click "Add Parameter"6. 
6. See that you just get a text input for name and value7. 

## Expected behavior

Ideally the name input would be a select input of the possible query parameters available.

## Is your portal managed or self-hosted?

Managed

## API Management service name

apim-iati-prod

## Environment

 - Operating system: [e.g., iOS] Mac OS 12.3
 - Browser: [e.g., Google Chrome, Safari] Chrome
 - Version: [e.g., 22] Version 100.0.4896.60 

## Additional context

Add any other context about the problem here, including screenshots.

![image](https://user-images.githubusercontent.com/60047271/161546668-bfa70b5c-3feb-4315-880c-7a48b3ecc840.png)

**Comment by ghost** [comment]
@nosvalds, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@nosvalds, thank you for requesting this feature.
